### PR TITLE
feat: report process MAC labels

### PR DESCRIPTION
**Pull request by dsseng**
# Pull Request

## What? (description)
Report process labels (typically used by Linux Security models)

## Why? (reasoning)i
This will be useful for debugging process access rights once we start implementing SELinux

## Acceptance

Please use the following checklist:

- [ ] you linked an issue (if applicable)
- [ ] you included tests (if applicable)
- [ ] you ran conformance (`make conformance`)
- [x] you formatted your code (`make fmt`)
- [x] you linted your code (`make lint`)
- [x] you generated documentation (`make docs`)
- [ ] you ran unit-tests (`make unit-tests`)

> See `make help` for a description of the available targets.

### CODE REVIEW

File: cmd/talosctl/cmd/talos/processes.go [hunk 1]
```diff
@@ -220,9 +220,11 @@ func processesOutput(ctx context.Context, c *client.Client) (output string, err
 				node = msg.Metadata.Hostname
 			}
 
+			fmt.Printf("`%s`\n", p.Label)
+
```
Comment (dsseng) [inline cmd/talosctl/cmd/talos/processes.go:224]:
```suggestion
```

File: cmd/talosctl/cmd/talos/processes.go [hunk 2]
```diff
@@ -190,7 +190,7 @@ func processesOutput(ctx context.Context, c *client.Client) (output string, err
 
 	s := []string{}
 
-	s = append(s, "NODE | PID | STATE | THREADS | CPU-TIME | VIRTMEM | RESMEM | COMMAND")
+	s = append(s, "NODE | PID | STATE | THREADS | CPU-TIME | VIRTMEM | RESMEM | LABEL | COMMAND")
```
Comment (dsseng) [inline cmd/talosctl/cmd/talos/processes.go:193]: Should we add a flag to only show these when wanted as typically users shouldn't need SELinux labels?

File: cmd/talosctl/cmd/talos/processes.go (hunk 1 above)
Comment (dsseng) [reply cmd/talosctl/cmd/talos/processes.go:224]: Left a debug line here...

**Comment by smira** [comment]
I have a PR coming later today to fork the docs for Talos 1.8, so we can start merging PRs with new features

File: cmd/talosctl/cmd/talos/processes.go (hunk 2 above)
Comment (smira) [reply cmd/talosctl/cmd/talos/processes.go:193]: I'd say fine to have a label always

File: cmd/talosctl/cmd/talos/processes.go [hunk 3]
```diff
@@ -220,9 +220,11 @@ func processesOutput(ctx context.Context, c *client.Client) (output string, err
 				node = msg.Metadata.Hostname
 			}
 
+			fmt.Printf("`%s`\n", p.Label)
+
 			s = append(s,
-				fmt.Sprintf("%12s | %6d | %1s | %4d | %8.2f | %7s | %7s | %s",
-					node, p.Pid, p.State, p.Threads, p.CpuTime, humanize.Bytes(p.VirtualMemory), humanize.Bytes(p.ResidentMemory), args))
+				fmt.Sprintf("%12s | %6d | %1s | %4d | %8.2f | %7s | %7s | %64s | %s",
```
Comment (smira) [inline cmd/talosctl/cmd/talos/processes.go:226]: 64 is like a lot

Comment (dsseng) [reply cmd/talosctl/cmd/talos/processes.go:226]: They're around 40 typically. The table shrinks anyway when column width isn't filled

File: internal/app/machined/internal/server/v1alpha1/v1alpha1_server.go [hunk 4]
```diff
@@ -1591,6 +1591,13 @@ func (s *Server) Processes(ctx context.Context, in *emptypb.Empty) (reply *machi
 			break
 		}
 
+		labelPath := fmt.Sprintf("/proc/%d/attr/current", info.Pid)
```
[the remark this answers is not in the document]
Comment (dsseng) [reply internal/app/machined/internal/server/v1alpha1/v1alpha1_server.go:1594]: sane, will do

**Pull request merged**